test/test_automl/test_estimators.py

```diff
@@ -21,9 +21,10 @@
 from autosklearn.automl import AutoMLClassifier, AutoML
 from autosklearn.util.backend import Backend, BackendContext
 from autosklearn.constants import BINARY_CLASSIFICATION
-from base import Base
 
 sys.path.append(os.path.dirname(__file__))
+from base import Base
```

Comment:
Shouldn't this be the correct order of these two lines of code?

Reply:
I guess so.